routers/api/v1/repo/issue_tracked_time.go

```diff
@@ -10,6 +10,14 @@ import (
 	api "code.gitea.io/sdk/gitea"
 )
 
+func convertTrackedTimes(trackedTimes []*models.TrackedTime) []*api.TrackedTime {
```

Comment:
`convertTrackedTimes` should also probably more specific for example `convertTrackedTimesToAPIFormat` otherwise it's hard to understand just from name what it does acutally

Reply:
Renamed to `trackedTimesToAPIFormat`